bftengine/src/bftengine/ReplicaConfigSerializer.cpp

```diff
@@ -90,6 +90,9 @@ void ReplicaConfigSerializer::serializeDataMembers(ostream &outStream) const {
   outStream.write((char *)&config_->autoPrimaryRotationTimerMillisec,
                   sizeof(config_->autoPrimaryRotationTimerMillisec));
 
+  // Serialize metricsDumpIntervalSeconds
+  outStream.write((char *)&config_->metricsDumpIntervalSeconds, sizeof(config_->metricsDumpIntervalSeconds));
```

Comment:
metricsDumpIntervalSeconds is the last parameter in the struct; please serialize/deserialize it as a last one saving the order.

Reply:
Done